at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -165,12 +167,12 @@ class GetFreeAtsign extends RegisterApiTask<RegisterApiResult<Map<String, String
 
     @Override
     public RegisterApiResult<Map<String, String>> run() {
-        System.out.println("Getting free atsign ...");
+        System.out.println("Fetching free atsign ...");
```

Comment:
replace sysout with logger

Reply:
at_java does not have a logger yet

Reply:
Yeah we should have a separate ticket to introduce a proper logger. Not for this PR though